tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -130,38 +108,58 @@ host: '<host>', port: 4000,user: '<username>', password: '<your_password>', data
 </div>
 </SimpleTab>
 
-## Where is the CA root path on my system?
+## Create and Reset Serverless database password
+
+When you connect to your cluster for the first time, click "Create Password" to automatically generate a password with a length of 16 characters, including numbers, uppercase and lowercase characters, and special characters. The password will be automatically embedded in the sample code for connecting to your cluster easily .
+
+You should copy the password after you create it, and only reset the password after leaving the connect page. Since the serverless cluster is accessed through the Internet,if you need to use the password elsewhere, every time you click on your cluster connection, you need to reset the password of your database cluster to ensure the security and compliance of the database password.
+
```

Comment:
This seems irrelevant to the topic of this article. We can put this description in Password authentication.

Reply:
I agree. Seems the modified version tells user the general way to connect to serverless tier. Not only about TLS.

Reply:
Let‘s discuss this content, as you know , Password authen just introduce how to set signin and signup for TiDB Cloud ,no focus on Database